staging/src/k8s.io/controller-manager/pkg/informerfactory/informer_factory.go

```diff
@@ -42,6 +43,18 @@ func (i *informerFactory) ForResource(resource schema.GroupVersionResource) (inf
 }
 
 func (i *informerFactory) Start(stopCh <-chan struct{}) {
+	trimFunc := func(obj interface{}) (interface{}, error) {
```

Comment:
@apelisse this code removes ManagedFields, PTAL.

Reply:
Ah, yes, that's what I missed, thanks :-)